packages/lib/src/interactions/AxialSelectionTool.tsx

```diff
@@ -26,30 +25,26 @@ function AxialSelectionTool(props: Props) {
   const { worldToData } = useVisCanvasContext();
   const camera = useThree((state) => state.camera);
 
-  const toAxialSelection = useCallback(
-    (selection: Selection): Selection => {
-      const { worldStartPoint, worldEndPoint } = selection;
-      const { bottomLeft, topRight } = getWorldFOV(camera);
-
-      const axialWorldStartPoint =
-        axis === 'x'
-          ? new Vector3(worldStartPoint.x, bottomLeft.y, 0)
-          : new Vector3(bottomLeft.x, worldStartPoint.y, 0);
-
-      const axialWorldEndPoint =
-        axis === 'x'
-          ? new Vector3(worldEndPoint.x, topRight.y, 0)
-          : new Vector3(topRight.x, worldEndPoint.y, 0);
-
-      return {
-        startPoint: worldToData(axialWorldStartPoint),
-        endPoint: worldToData(axialWorldEndPoint),
-        worldStartPoint: axialWorldStartPoint,
-        worldEndPoint: axialWorldEndPoint,
-      };
-    },
-    [axis, camera, worldToData]
-  );
+  function toAxialSelection(selection: Selection): Selection {
+    const [worldStart, worldEnd] = selection.world;
+    const { bottomLeft, topRight } = getWorldFOV(camera);
+
+    const axialWorldSelection: Rect3 =
```

Comment:
One ternary that returns a `Rect3` allows me to map through it below to convert it to data space.